src/mixins/sliding-menu.js

```diff
@@ -27,13 +27,17 @@ export default {
   methods: {
     ...mapActions(['toggleSlidingMenu']),
 
-    toggleClickListener(isOn) {
+    toggleEventListener(eventName, isOn) {
       window[`${isOn ? 'add' : 'remove'}EventListener`](
-        'click',
+        eventName,
         this.onWindowClick,
         true
       );
     },
+    toggleClickListener(isOn) {
```

Comment:
not sure about `toggle` `isOn` is make it on or toggle it off from on
just use sth like `isAdd` / `isRemove`?

Reply:
Updated